spring-integration-file/src/main/java/org/springframework/integration/file/config/FileParserUtils.java

```diff
@@ -118,7 +124,7 @@ static void configureFilter(BeanDefinitionBuilder synchronizerBuilder, Element e
 			}
 			if (count != 1) {
 				parserContext.getReaderContext().error("at most one of 'filename-pattern', " +
-						"'filename-regex', 'filter' or 'filter-expression' is allowed on remote file inbound adapter",
+								"'filename-regex', 'filter' or 'filter-expression' is allowed on remote file inbound adapter",
```

Comment:
I used the reformatting tool manually (Ctrl+Alt+L) and it is applied for the whole code.

Again: I don't know how to make it only for the touch code and don't know how to not miss violations in other lines.

I guess if I have the proper setting for these kind of indents, even if I perform that reformatting, this line of code wouldn't be touch and we wouldn't have such a long discussion....